packages/scene-composer/src/components/panels/SceneNodeInspectorPanel.tsx

```diff
@@ -131,7 +131,11 @@ export const SceneNodeInspectorPanel: React.FC = () => {
         >
           <FormField label={intl.formatMessage({ defaultMessage: 'Name', description: 'Form field label' })}>
             <TextInput
-              data-testid={`ip-${selectedSceneNode.components[0].ref}`}
+              data-testid={
+                selectedSceneNode.components && selectedSceneNode.components.length > 0
```

Comment:
nit-pick: This could be simplified with optional chaining. That's on me. I should have called it out earlier. 

`selectedSceneNode?.components.length > 0`

Reply:
Done